### PR TITLE
Updated "Using Add-ons API" as private preview

### PR DESCRIPTION
**Pull request by ashishnirvana1**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-1704

Made a few changes to indicate that this API is available in private preview hence not visible to everyone.  

Writer/Editor pass:
vchapel

### CODE REVIEW

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ To use the Microsoft Edge Add-ons API, you need to enable the API for your proje
 ## Retrieving the access token
 
 > [!NOTE]
-> The Microsoft Edge Add-ons API is not yet available for testing.
+> The Microsoft Edge Add-ons API is currently in private preview.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:54]:
```suggestion
> The Microsoft Edge Add-ons API is currently available only for participants of the private preview.
```

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatic
 To use the Microsoft Edge Add-ons API, you need to enable the API for your project in the Microsoft Partner Center.
 
 > [!NOTE]
-> The **Publish APIs** UI is not yet present at Partner Center.
+> The **Publish APIs** is in private preview and the UI is not yet present at Partner Center for everyone.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:39]:
```suggestion
> The **Publish API** page is only present at Partner Center for participants of the private preview.
```

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.date: 08/19/2021
 ---
-# Using the Microsoft Edge Add-ons API (under development)
+# Using the Microsoft Edge Add-ons API (in private preview)
 
 > [!NOTE]
-> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
+> The Microsoft Edge Add-ons API is currently in private preview, and the Publish APIs page is not yet available at Partner Center for everyone.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:13]: The page title at Partner Center is **Publish API** not **Publish APIs**; see screenshot at https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-1694#before-you-begin
```suggestion
> The Microsoft Edge Add-ons API is currently in private preview.  The Publish API page is available at Partner Center only for participants of the private preview.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here aren't exhaustive, and are subject to change.
```

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/extensions-chromium/publish/api/using-addons-api.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.date: 08/19/2021
 ---
-# Using the Microsoft Edge Add-ons API (under development)
+# Using the Microsoft Edge Add-ons API (in private preview)
 
 > [!NOTE]
-> This article is a Request for Comments.  The Microsoft Edge Add-ons API is not yet available for testing, and the Publish APIs page is not yet available at Partner Center.  The Microsoft Edge Add-ons API is under active development and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here are not exhaustive and are subject to change.
+> The Microsoft Edge Add-ons API is currently in private preview.  The **Publish API** page is present at Partner Center only for participants of the private preview.  The Microsoft Edge Add-ons API is under active development, and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here aren't exhaustive, and are subject to change.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/api/using-addons-api.md:13]:
```suggestion
> This article is a Request for Comments.  The Microsoft Edge Add-ons API is currently in private preview.  The **Publish API** page is present at Partner Center only for participants of the private preview.  The Microsoft Edge Add-ons API is under active development, and the roadmap continues to evolve based on market changes and customer feedback.  The plans outlined here aren't exhaustive, and are subject to change.
```
PR 1704 & 1705 should both have "This article is a Request for Comments.  ", or neither should have that statement.

**Review by vchapel — APPROVED** [review]
LGTM

**Pull request merged**